sql/moz-fx-data-shared-prod/firefox_desktop_derived/urlbar_events_daily_v1/query.sql

```diff
@@ -0,0 +1,110 @@
+WITH
+  temp_unnested AS (
+  SELECT
+    submission_date,
+    event_id,
+    event_action,
+    res.product_result_type AS product_result_type,
+    normalized_channel,
+    normalized_country_code,
+    pref_fx_suggestions,
+    pref_sponsored_suggestions,
+    is_terminal,
+    CASE
+      WHEN (product_selected_result = res.product_result_type AND event_action = 'engaged' AND is_terminal = TRUE) THEN 1
+    ELSE
+    0
+  END
+    AS is_clicked,
+    CASE
+      WHEN (product_selected_result = res.product_result_type AND event_action = 'annoyance') THEN 1
+    ELSE
+    0
+  END
+    AS is_annoyed,
+  FROM
+    `mozdata.firefox_desktop.urlbar_events`
+  CROSS JOIN
+    UNNEST (results) AS res
+  WHERE
+    submission_date = @submission_date ),
+  temp_session AS (
+  SELECT
+    submission_date,
+    event_id,
+    product_result_type,
+    is_terminal,
+    mozdata.udf.mode_last(ARRAY_AGG(normalized_channel)) AS normalized_channel,
```

Comment:
Since these fields should all be the same when grouping by `event_id`, you can use `ANY_VALUE(normalized_channel)` to simplify picking one value from the group.